pkg/client-sdk/client/client.go

```diff
@@ -263,3 +274,125 @@ type AddressEvent struct {
 	SpentVtxos []Vtxo
 	Err        error
 }
+
+// IndexerService types
+
+type PageRequest struct {
+	Size  int32
+	Index int32
+}
+
+type PageResponse struct {
+	Current int32
+	Next    int32
+	Total   int32
+}
+
+type IndexerNode struct {
+	Txid       string
+	ParentTxid string
+	Level      int32
+	LevelIndex int32
+}
+
+type Batch struct {
+	TotalBatchAmount   uint64
+	TotalForfeitAmount uint64
+	TotalInputVtxos    int32
+	TotalOutputVtxos   int32
+	ExpiresAt          int64
+	Swept              bool
+}
+
+type CommitmentTxInfo struct {
+	StartedAt int64
+	EndedAt   int64
+	Batches   map[uint32]*Batch
+}
+
+type VtxoTreeResponse struct {
+	VtxoTree []IndexerNode
+	Page     PageResponse
+}
+
+type ForfeitTxsResponse struct {
+	Txs  []string
+	Page PageResponse
+}
+
+type ConnectorsResponse struct {
+	Connectors []IndexerNode
+	Page       PageResponse
+}
+
+type IndexerVtxo struct {
+	Outpoint  Outpoint
+	CreatedAt int64
+	ExpiresAt int64
+	Amount    uint64
+	Script    string
+	IsLeaf    bool
+	IsSwept   bool
+	IsSpent   bool
+	SpentBy   string
```

Comment:
Right now, this propoerty being set is only exposed through `SubscribeForAddressesResponse`